qlib/strategy/base.py

```diff
@@ -210,6 +225,9 @@ def __init__(
         super(RLStrategy, self).__init__(outer_trade_decision, level_infra, common_infra, **kwargs)
         self.policy = policy
 
+    def generate_trade_decision(self, execute_result: list = None) -> BaseTradeDecision:
+        pass  # TODO: to be implemented
```

Comment:
Why not raising a NotImplementError  just keep it abstract?

Reply:
Reverted. Similar to the previous comment.

Reply:
I think not implementing `generate_trade_decision` means that  `RLStrategy` is still abstract strategy